coretex/utils/process.py

```diff
@@ -45,13 +45,14 @@ def command(
     ignoreStdout: bool = False,
     ignoreStderr: bool = False,
     shell: bool = False,
-    check: bool = True
+    check: bool = True,
+    cwd: str = str(Path(__file__).parent)
```

Comment:
Update `cwd` type to `pathlib.Path`. The value of `cwd` should default to `Path.cwd()`